### PR TITLE
do not treat warnings as errors in Java v1.8_191-b26-OpenJDK (POWER)

### PR DESCRIPTION
**Pull request by terjekv**
(created using `eb --new-pr`)

`readdir_r` is deprecated. https://bugs.java.com/bugdatabase/view_bug.do?bug_id=JDK-8179887

### CODE REVIEW

**Comment by terjekv** [comment]
@Flamefire Can you look at this and give it a whirl? It was required on RHEL8.3 on Power.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
taurusml29 - Linux RHEL 7.6, POWER, 8335-GTX, Python 2.7.5
See https://gist.github.com/a21104bd78d74379aade734db89f51e6 for a full test report.

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb [hunk 1]
```diff
@@ -61,7 +61,7 @@ preconfigopts += 'mv ../jdk-* jdk && '
 preconfigopts += 'mv ../hotspot-* hotspot && '
 preconfigopts += 'mv ../nashorn-* nashorn && '
 preconfigopts += 'chmod u+x configure && '
-configopts = '--with-cacerts-file=/etc/pki/ca-trust/extracted/java/cacerts '
+configopts = '--with-extra-cxxflags="-Wno-error" --with-extra-cflags="-Wno-error" --with-cacerts-file=/etc/pki/ca-trust/extracted/java/cacerts '
```
[the remark this answers is not in the document]
Comment (terjekv) [reply easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb]: Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/11755/commits/38b4e5b15fe352b8d230bae6dde2b3f1ae773f4d. Thanks!

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node376.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c26c9481253a318f3c42da7455c39ecd for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node128.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/aab37d62dead5ba627ce737c90833be1 for a full test report.

**Comment by terjekv** [comment]
```
configure: error: Could not find all X11 headers (shape.h Xrender.h XTest.h Intrinsic.h). You might be able to fix this by running 'sudo yum install libXtst-devel libXt-devel libXrender-devel'.
configure exiting with result code 1
```
Why haven't we seen this before?

**Comment by lexming** [comment]
@terjekv my test env is the best :sunglasses:

**Comment by terjekv** [comment]
I thought that the OSDEP `('libXtst-devel', 'libxtst-dev'),` was supposed to fix that? Hm.

**Comment by lexming** [comment]
I do not have `libXtst-devel` installed and EB failed to tell me that.

**Comment by terjekv** [comment]
Okay, that's _bad_.

**Comment by lexming** [comment]
@terjekv I found the issue, I always build with `--force` and that build option disables the validation of OS dependencies

**Comment by terjekv** [comment]
`--rebuild`?

**Comment by boegel** [comment]
@lexming This `Java` easyconfig is only relevant for POWER, no? See `Java-1.8.eb`.

**Comment by boegel** [comment]
> `--rebuild`?

Indeed, `--force` is equivalent with `--rebuild --ignore-osdeps`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild1.novalocal - Linux centos linux 8.2.2004, POWER, IBM pSeries (emulated by qemu) (power8le), Python 3.6.8
See https://gist.github.com/fd81c1be3007055c412fbab863a9789f for a full test report.

**Comment by lexming** [comment]
@boegel I'm so bad! how did I not think about `--rebuild` :smile:
Regarding the architecture, this is used for POWER just because the lack of support in Oracle Java, but OpenJDK still works in x86 systems as well.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login4.phoenix.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/e5b61f16fca153583661f37833dbde2a for a full test report.

**Comment by lexming** [comment]
Going in, thanks @terjekv !

**Pull request merged**